Src/WitsmlExplorer.Api/Workers/CreateLogObjectWorker.cs

```diff
@@ -0,0 +1,157 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Serilog;
+using Witsml;
+using Witsml.Data;
+using Witsml.Extensions;
+using Witsml.Query;
+using Witsml.ServiceReference;
+using WitsmlExplorer.Api.Jobs;
+using WitsmlExplorer.Api.Models;
+using WitsmlExplorer.Api.Services;
+using System;
+using System.Threading;
+using Witsml.Data.Measures;
+using System.Collections.Generic;
+
+namespace WitsmlExplorer.Api.Workers
+{
+    public interface ICreateLogObjectWorker
+    {
+        Task<(WorkerResult, RefreshWellbore)> Execute(CreateLogObjectJob job);
+    }
+
+    public class CreateLogObjectWorker : ICreateLogObjectWorker
```

Comment:
We already have a `CreateLogWorker` in this project, which creates an empty log prepared for data added by `UpdateLogWorker`(?).

There should be some work done to either merge these two or distinguish them better by name as the current code will be confusing to use